src/data/faq_de.json

```diff
@@ -409,7 +409,9 @@
                                 "anchor": "rec_cert",
                                 "active": false,
                                 "textblock": [
-                                    "Ab der Version 2.5 der Corona-Warn-App werden Genesenenzertifikate unterstützt. Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeit entfällt bei allen Impfstoffen außer bei Johnson & Johnson."
+                                    "<b>Aktualisiert am 30. März 2022</b>",
+                                    "Ab der Version 2.5 der Corona-Warn-App werden Genesenenzertifikate unterstützt. Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeit entfällt bei allen Impfstoffen außer bei Johnson & Johnson.",
+                                    "Ab Version 2.20 werden auf Genesenenzertifikaten keine Gültigkeitsdaten mehr angezeigt. Da die Gültigkeit der Zertifikate von lokalen Regeln abhängig ist, wird stattdessen nun das Datum des ersten positiven Tests angezeigt, das dem Zertifikat zu Grunde liegt."

```

Comment:
Duden recommends "zugrunde" instead of "zu Grunde"

https://www.duden.de/rechtschreibung/zugrunde

Reply:
Thanks!